### PR TITLE
Fix fonttbl bottleneck

### PR DESCRIPTION
**Pull request by kebsen**
Hey @joshy,

Thanks for maintaining what seems to be one of the only RTF parsers for Python.

I used this package and noticed that it takes some time to parse large RTF files. We are talking about more than 300 pages sometimes, which took over 3 minutes on my machine. I investigated this issue and it seems that the regex matching the font table entries is quite expensive to run against the entire text.

To address this, I added a helper method to extract the font table out of the text by simply counting opening and closing braces. The original regex is then used to search only this part of the text. You could probably parse the font table entries directly when counting braces, but I am not that deep in the RTF-spec at all, so I leave this to the experts😉.

Btw., one could probably also use similar logic to solve #69.

I also added a brief hint for developers on Windows as I ran into encoding issues while executing pytest.

### CODE REVIEW

**Comment by joshy** [comment]
@kebsen Thanks, I am currently on vacation, will check when I am back

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -45,6 +45,8 @@ For verbose output:
 pytest -v
 ```
 
+On Windows, make sure Python uses UTF-8, for example by setting the environment variable `PYTHONUTF8=1`.
```
Comment (stevengj) [inline DEVELOPMENT.md]: It seems like a bad idea to rely on a global environment variable, rather than setting the charset when opening a file.

Comment (kebsen) [reply DEVELOPMENT.md]: Just to clarify, are you suggesting updating all test cases to explicitly open text files with UTF-8 encoding, as is already done here: https://github.com/joshy/striprtf/blob/b2e88aaff11aabb9d67f02cd72e98a0213fcd1b7/tests/test_ansicpg1250.py? I completely agree that this is a better solution to my problems.

**Comment by joshy** [comment]
@kebsen Hey, can you share some of those rtfs?

**Comment by kebsen** [comment]
> @kebsen Hey, can you share some of those rtfs?

@joshy Hey, sorry for the late response, I was on vacation as well. Here is a [large rtf](https://github.com/user-attachments/files/31138134/large_rtf.rtf) I also used for testing. I did not add a test case (yet), because it does not cover any edge case like in other issues and is only a performance improvement. I can add one with this file if you want, though.

**Comment by joshy** [comment]
@kebsen Thanks for the PR, the issue is now fixed with version 0.0.33
